### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ostUp**
[DEMO LINK](https://ostUp.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,61 +1,118 @@
...
+      user: getUser(userId),
+    };
+
+    if (userId && title.trim()) {
+      todos.push(newTodo);
+    } else {
+      setTitleError(title.trim() === '');
+      setUserIdError(userId === 0);
+
+      if (title.trim() === '') {
```
Comment (DimaKobzar7) [inline src/App.tsx:58]:
```if (userId && title.trim()) {
      todos.push(newTodo);
    } else {
      setTitleError(title.trim() === '');
      setUserIdError(userId === 0);

      if (title.trim() === '') {
        setTitle('');
      }

      return;
    }```

Comment (DimaKobzar7) [reply src/App.tsx:58]: nested conditions "if" is not the best solution. Try to rewrite so that inside the "if" block you don't have another "if" block

**Review by DimaKobzar7 — APPROVED** [review]

**Comment by DimaKobzar7** [comment]
It`s look better. Good work

File: src/App.tsx [hunk 2]
```diff
@@ -1,61 +1,116 @@
...
+
+    setUserId(+value);
+  };
+
+  const handleSubmitForm = (event: React.ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const maxId = Math.max(0, ...todos.map(({ id }) => id));
+
+    if (title.trim() === '' || userId === 0) {
```
Comment (AlinaMatuschak) [inline src/App.tsx:46]:
```suggestion
    if (!title.trim() || !userId) {
```

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**